commit 3343101d993edf10f9782e390a600ca568b840f7
Author: Alan F <...>
Date:   Fri Apr 25 19:32:19 2014 +0100

    issue 260 - present (bring to front) the torrent add options dialog to the user when it is opened.

--- src/trg-main-window.c.orig	2014-04-25 22:23:59.831553775 +0200
+++ src/trg-main-window.c	2014-04-25 22:26:22.318600733 +0200
@@ -523,6 +523,7 @@
                                        filesList);
 
         gtk_widget_show_all(GTK_WIDGET(dialog));
+	gtk_window_present(GTK_WINDOW(dialog));
     } else {
         struct add_torrent_threadfunc_args *args =
             g_new0(struct add_torrent_threadfunc_args, 1);
--- src/trg-torrent-add-dialog.c.orig	2014-04-25 22:24:47.971907498 +0200
+++ src/trg-torrent-add-dialog.c	2014-04-25 22:33:19.626130963 +0200
@@ -944,8 +944,8 @@
             TrgTorrentAddDialog *dialog = trg_torrent_add_dialog_new(win,
                                                                      client,
                                                                      l);
-
             gtk_widget_show_all(GTK_WIDGET(dialog));
+            gtk_window_present(GTK_WINDOW(dialog));
         } else {
             struct add_torrent_threadfunc_args *args =
                 g_new0(struct add_torrent_threadfunc_args, 1);
